src/trufflecon2020.html

```diff
@@ -404,15 +497,36 @@ <h3 class="mb-1">Patrick Collins</h3>
         <p class="mb-0">Developer Advocate</p>
         <p class="mb-5 font-italic">Chainlink</p>
       </div>
-  
 
+      <div class="col-6 col-lg-3 iso-item official">
+        <img class="rounded-circle mb-2 img-fluid" src="/img/trufflecon/2020speakers/richard-meissner.jpg" alt="Richard Meissner" />
+        <h3 class="mb-1">Richard Meissner</h3>
+        <p class="mb-0">Tech Lead</p>
+        <p class="mb-5 font-italic">Gnosis</p>
+      </div>
+
+      <div class="col-6 col-lg-3 iso-item official">
+        <img class="rounded-circle mb-2 img-fluid" src="/img/trufflecon/2020speakers/shane-fontaine.jpg" alt="Shane Fontaine" />
+        <h3 class="mb-1">Shane Fontaine</h3>
+        <p class="mb-0">Co-Founder</p>
+        <p class="mb-5 font-italic">Authereum</p>
+      </div>
+
+      <div class="col-6 col-lg-3 iso-item official">
+        <img class="rounded-circle mb-2 img-fluid" src="/img/trufflecon/2020speakers/solange-gueiros.jpg" alt="Solange Gueiros" />
+        <h3 class="mb-1">Solange Gueiros</h3>
+        <p class="mb-0">Developer</p>
+        <p class="mb-5 font-italic">RSK</p>
+      </div>
+  
+<!--
       <div class="col-6 col-lg-3 iso-item official">
         <img class="rounded-circle mb-2 img-fluid" src="/img/trufflecon/2020speakers/rob-steiner.jpg" alt="Rob Steiner" />
         <h3 class="mb-1">Rob Steiner</h3>
         <p class="mb-0">Head of Distributed Ledger Technologies, Wireless Innovation</p>
         <p class="mb-5 font-italic">Dish</p>
       </div>
-
+-->
```

Comment:
Is there a reason for commenting this out?

Reply:
Yeah, Rob doesn't work with Dish anymore so Liz wanted to pull him... there may still be a chance he would still speak though, so I decided to comment it out.